src/app.multitenancy.test.ts

```diff
@@ -0,0 +1,419 @@
+import express from 'express';
+import { clone, BatchReadWriteResponse } from 'fhir-works-on-aws-interface';
+import { generateServerlessRouter } from './app';
+import r4FhirConfigGenericMultiTenancy from '../sampleData/r4FhirConfigGenericMultiTenancy';
+import ElasticSearchService, { setExpectedSearchSet } from './router/__mocks__/elasticSearchService';
+import DynamoDbDataService, { setExpectedCreateResourceId } from './router/__mocks__/dynamoDbDataService';
+import DynamoDbBundleService, { setExpectedBundleEntryResponses } from './router/__mocks__/dynamoDbBundleService';
+import AuthorizationService, { setExpectedTokenDecoded } from './router/__mocks__/authorizationService';
+import validPatient from '../sampleData/validV4Patient.json';
+
+// eslint-disable-next-line import/no-unresolved
+const resourceType: string = 'Patient';
+const resourceId: string = '12345';
+const defaultTenantId: string = 'DEFAULT';
+const specificTenantId: string = '915b76f7-8744-4010-bd31-a1e4c0d9fc64';
+const wrongTenantId: string = '1234567890';
+const anyTenantId: string = '09876543';
+
+const request = require('supertest');
```

Comment:
should be an import as the rest

Reply:
The AWS guys also uses require('supertest') in their tests, see app.test.ts

Reply:
We don't need to copy something if its wrong :)  if its a global require, then it should be converted into an import (in webstorm when selecting the line, right click -> show context actions -> convert require to import). 

Don't forget to run the tests just to make sure

Reply:
Then the following dependencies needs to be added:
     "@types/supertest": "^2.0.12",
    "@types/superagent": "4.1.13",  <= Workaround: Here a fixed version needs to set, because  the 4.1.14 has an export buffer form Blob bug
and "@types/node": "^12 needs to be updated to "@types/node": "^14.0.1",

I guess the AWS guys did that way to avoid these dependencies.

Reply:
Hummm, I see, then typescript also tries to import the types with the import command. ok, then lets leave it as they did :)